spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
Well, we do advertise that we follow the Spring Cloud code conventions on both https://github.com/spring-cloud/spring-cloud-stream and https://github.com/spring-cloud/spring-cloud-dataflow#contributing, which link back to https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-docs/src/main/asciidoc/appendix-contributing.adoc, which link back to the Spring Framework code conventions. It doesn't help that the eclipse settings in Spring Cloud Commons are even less standard than anything else: https://github.com/spring-cloud/spring-cloud-commons/blob/master/spring-cloud-commons/eclipse/org.eclipse.jdt.ui.prefs#L67. I guess we do need to be consistent within the Spring Cloud group of project. So, under the circumstances, I think that adopting the Spring Framework rules would give us a reference point. I am willing to adjust, but let's be consistent. Also, I have noticed that Spring Cloud Task has adopted  the Spring Framework rules as well (including imports). cc @mminella @dsyer